app/src/ui/components/ThemeToggle.module.css

```diff
@@ -0,0 +1,42 @@
+.position {
+  position: absolute;
+  top: 0;
+  right: 50px;
+}
+
+.checkbox {
+  opacity: 0;
+}
+
+.checkbox-label {
+  background-color: var(--second-color);
+  width: 40px;
+  height: 16px;
+  border-radius: 50px;
+  position: relative;
+  padding: 5px;
+  cursor: pointer;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border: 1px solid #646464;
+}
+
+.checkbox-label .ball {
+  background-color: #fff;
+  width: 22px;
+  height: 22px;
+  position: absolute;
+  left: 2px;
+  top: 2px;
+  border-radius: 50%;
+  transition: transform 0.2s linear;
+}
+
+.checkbox:checked + .checkbox-label .ball {
+  transform: translateX(24px);
+}
+
+.light {
+  background-color: rgb(127, 98, 177);
+}
```

Comment:
please, use the mechanism we just implemented. 

do the colors for background via `composes` just like you did with all another components